### ISSUE TITLE
CSP 'script-src' directive is violated by Function constructor

### ISSUE THREAD

**Issue by dimovpetar** [ajv-validator/ajv#1640]
Hello,

The usage of `Function` constructor violates the 'script-src' directive. Right now, the only workaround is to disable it by setting it's value to `unsafe-eval`.
Line: https://github.com/ajv-validator/ajv/blob/f54f1b7f463b8b71e7e56b9bd2684271810fe977/lib/compile/index.ts#L171

More information about [Unsafe eval expessions](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#unsafe_eval_expressions).
Are there any plans to refactor it and enable the CSP policy or is there any workaround?

**Comment by epoberezkin** [comment]
Ajv is a compiler; if you compile schemas at run time it has to use Function constructor. See https://ajv.js.org/guide/environments.html#browsers and https://ajv.js.org/security.html#content-security-policy

**Comment by dimovpetar** [comment]
Hello @epoberezkin ,
Thanks for the answer. I've followed https://github.com/ajv-validator/ajv/blob/master/docs/standalone.md but I still have some questions:

- Are there any options for the module syntax? Currently it uses CommonJS. I'd like to use it in browser
- `const standaloneCode = require("ajv/dist/standalone")` didn't work, because standaloneCode is not a function. I had to use `const standaloneCode = require("ajv/dist/standalone").default`. Is this expected?
